library/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateAksStep.java

```diff
@@ -145,6 +163,23 @@ private KubernetesCluster createAks(FlightContext context) {
     return aks;
   }
 
+  private WithAttach addAutoscale(WithAttach aksPartial) {
```

Comment:
1. does autoscale enable it cluster level or node pool level? If i can just call it once that'd be great.
2. Code smells are complaining because I didn't type the generic, but the two places that call them are different types (one is `KubernetesCluster.DefinitionStages.WithCreate` and the other is `KubernetesCluster.Update`. Do i get rid of the function and just duplicate the code?

Reply:
I think we should avoid using raw types. Potentially such code is unsafe.